### PR TITLE
Upgraded stm32f7 CubeMX to version 1.11.0 (latest)

### PR DESCRIPTION
**Pull request by mlampert**
I ran into some bugs with the currently used version of CubeMX for the STM32F7, whatever version that was. Upgraded to the latest version which has the issues fixed.

### CODE REVIEW

**Comment by utzig** [comment]
What bug did you run into?

**Comment by mlampert** [comment]
LL_TIM_EnableUpdateEvent & LL_TIM_DisableUpdateEvent have the logic reversed.

File: hw/mcu/stm/stm32f7xx/pkg.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ pkg.ign_files:
 
 pkg.ign_dirs:
     - "Device"
+    - "Examples"
```
Comment (utzig) [inline hw/mcu/stm/stm32f7xx/pkg.yml]: Please remove those directories instead

**Comment by utzig** [comment]
Thanks!

**Pull request merged**